app/views/admin/articles/_form.html.erb

```diff
@@ -23,7 +23,7 @@
 
     <!-- TEMP: move to partial -->
     <%# TEMP: enable for alpha tolerant users, only on unpublished articles %>
-    <% if (1..4).include?(current_user.id) && @article.draft? %>
+    <% if Current.user.id == 1 && @article.draft? %>
```

Comment:
just checking, this is intentionally more restrictive?

Reply:
Yeah. It's just me for now. Until the feature is actually working. 

Good catch. 👍🏻